content/es/docs/reference/glossary/controller.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Controller
```

Comment:
```suggestion
title: Controladores
```
(compare https://kubernetes.io/pt/docs/concepts/architecture/controller/)

Reply:
```suggestion
title: Controlador
```

Thanks for the suggestion, going with the singular in Spanish.